src/styles/variables.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --text-size-secondary: 10px;
+  --text-weight-secondary: 400;
```

Comment:
It's a default font-weight value, so it can be removed

Reply:
done